source/anti-patterns/language.md

```diff
@@ -116,7 +106,7 @@ But that is still problematic, unless:
 2. you control the shell environment via your source code, setting `$NIX_PATH` via `nix-shell` or NixOS options.
 
 :::{note}
-We recommend to avoid using search paths and to disable channels by permanently setting `NIX_PATH=` to be empty.
+We recommend to avoid using extra search paths and to switch to flakes and disable channels by permanently setting `NIX_PATH=` to be empty.
```

Comment:
Again, switching to flakes should be supported by authoritative material on how to do that, otherwise it will just confuse people.

```suggestion
We recommend to avoid using search paths and disable channels by permanently setting `NIX_PATH=` to be empty.
```

The docs team decided "[not to drag people into unstable features](https://github.com/NixOS/nix.dev/issues/281#issuecomment-1198128399)" and instead build up Nix 3 and Flakes on a separate branch.

Reply:
Agreed, we shouldn't recommend flakes